packages/permissions/src/components/restricted-by-permissions/restricted-by-permissions.spec.tsx

```diff
@@ -3,29 +3,42 @@ import { ApplicationContextProvider } from '@commercetools-frontend/application-
 import { render } from '@testing-library/react';
 import RestrictedByPermissions from './restricted-by-permissions';
 
-type TPermissions = {
-  [key: string]: boolean;
+type TAllAppliedPermission = {
+  name: string;
+  value: boolean;
 };
-type TActionRight = {
-  [key: string]: boolean;
+type TAllAppliedActionRight = {
+  name: string;
+  value: boolean;
+  group: string;
 };
-type TActionRights = {
-  [key: string]: TActionRight;
+type TAllAppliedMenuVisibility = {
+  name: string;
+  value: boolean;
+};
+type TAllAppliedDataFence = {
+  __typename: 'StoreDataFence';
+  value: string;
+  group: string;
+  name: string;
+  type: string;
 };
-type TActionRightName = string;
-type TActionRightGroup = string;
 type TDemandedActionRight = {
-  group: TActionRightGroup;
-  name: TActionRightName;
+  group: string;
+  name: string;
 };
 
 const testRender = ({
-  actualPermissions = { canManageProjectSettings: true },
-  actualActionRights = {},
+  allAppliedPermissions = [{ name: 'canManageProjectSettings', value: true }],
+  allAppliedActionRights = [],
+  allAppliedMenuVisibilities = [],
+  allAppliedDataFences = [],
   component,
 }: {
-  actualPermissions?: TPermissions;
-  actualActionRights?: TActionRights;
+  allAppliedPermissions?: TAllAppliedPermission[];
+  allAppliedActionRights?: TAllAppliedActionRight[];
+  allAppliedMenuVisibilities?: TAllAppliedMenuVisibility[];
+  allAppliedDataFences?: TAllAppliedDataFence[];
```

Comment:
Will this result in a breaking change for `test-utils`? No right?

Reply:
Ah right, I forgot about that.

Hmm well in theory yes...maybe we can be backwards compatible and denormalize the value to be passed to the context provider.

Reply:
Yeah I think we might have to while logging something maybe. Bit of a annoying side-effect of this.

Reply:
Yeah, I didn't think about this... 🙃

Reply:
https://github.com/commercetools/merchant-center-application-kit/pull/1064/commits/a1f3854ebb73685f2f1b32e4a5a0e0dfd6fb648c